### PR TITLE
*: upgrade kvproto

### PR DESCRIPTION
**Pull request by NingLin-P**
Signed-off-by: linning <linningde25@gmail.com>

### What is changed and how it works?

What's Changed:

Upgrade kvproto to https://github.com/pingcap/kvproto/commit/5a0d6e96603c0f9ee7fa09ae6a0750125ab757b5

### Check List 

Tests 

- Manual test (add detailed scripts or steps below)

`make clippy`

### Release note 
- No release note

### CODE REVIEW

File: components/cdc/src/errors.rs [hunk 1]
```diff
@@ -65,8 +71,8 @@ impl Error {
             ))))
             | Error::Txn(TxnError(box TxnErrorInner::Mvcc(MvccError(
                 box MvccErrorInner::Engine(EngineError(box EngineErrorInner::Request(e))),
-            ))))
-            | Error::Request(e) => e,
+            )))) => e,
+            Error::Request(e) => e.as_ref().clone(),
```
Comment (BusyJay) [inline components/cdc/src/errors.rs:75]: How about `| Error::Request(box e) => e`?

**Review by BusyJay — COMMENTED** [review]
Rest LGTM

File: components/cdc/src/errors.rs (hunk 1 above)
Comment (NingLin-P) [reply components/cdc/src/errors.rs:75]: `e` is `Box<ErrorHeader>` not `ErrorHeader`

**Review by BusyJay — APPROVED** [review]

**Review by nolouch — APPROVED** [review]
lgtm

**Review by nolouch — APPROVED** [review]

**Pull request merged**